src/dos/drive_local.cpp

```diff
@@ -169,6 +169,16 @@ std::unique_ptr<DOS_File> localDrive::FileOpen(const char* name, uint8_t flags)
 		fallback_to_readonly = true;
 	}
 
+#if defined(WIN32)
+	// HACK: Always try to open host files in r/w mode on Windows since
```

Comment:
I'd remove "HACK" from the comment. As I see it, this is a proper mapping of the quirky DOS behaviour to the host filesystem. Probably I'd even prefer doing this on all OSes, not just Windows.